demos/src/demo.1.mustache

```diff
@@ -0,0 +1,3 @@
+<meter class="o-meter" aria-label="demo meter {{meterValue}}" data-o-component="o-meter" min="0" max="100" value="{{meterValue}}">
```

Comment:
We just need to update the markup in the README with the aria-label. Otherwise this is great and ready to go :D

Reply:
ah yes! of course! thank you!